tests/payloads/github.headers.txt

```diff
@@ -0,0 +1,7 @@
+Request URL: https://openedx-webhooks.herokuapp.com/github/pr
```

Comment:
I don't see this file (or github.json) referenced anywhere, and there are no new tests added.  Are these for a different pull request?